v0/destinations/googlesheets/transform.js

```diff
@@ -9,6 +9,7 @@ const getTabName = event => {
 
   return (
     (message.event ? hashMap[message.event.toLowerCase()] : null) ||
+    (message.name ? hashMap[message.name.toLowerCase()] : null) ||
```

Comment:
Added support for `message.name` for mapping to Tab in sheets (WDYT @ruchiramoitra )
Example event like
page call with a name `Home Page` to be mapped to a specific tab so in scenarios where `event` property is not present but `name` we will consider `name` for mapping the event to a Specific Sheet tab. Thoughts?